### DISCUSSION TITLE
Throw your ideas on hypertrons-crx design here!

### DISCUSSION THREAD

**Discussion by xiaoya-yaya** [hypertrons/hypertrons-crx#56] [Ideas]
@wengzhenjie @zhuxiangning

**Comment by frank-zsy** [comment]
I assume notification is very important for the first edition because we may update the project very often in the future and we need to have a way to notify users to update the extension to get new features.

**Reply by heming6666** [reply]
+1.

**Reply by heming6666** [reply]
**Update :** Thanks @frank-zsy for this great idea. This feature has been implemented in recent PRs (#82, #83, #94).

**Comment by tyn1998** [comment]
As a user, currently I find it hard to identify 
- "this project" node in Project Correlation Network
- "this developer" node in Developer Collabration Network

Maybe we can assign a strong color to this two nodes. For example, we can modify ["itemstyle"](https://echarts.apache.org/zh/option.html#series-graph.data) of a particular node to change its style in Echarts.

**Reply by heming6666** [reply]
It's a good idea.
